[2024-11-03T04:55:19.555+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-03T04:55:19.563+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.create_table_user_review manual__2024-11-03T04:55:16.752730+00:00 [queued]>
[2024-11-03T04:55:19.568+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.create_table_user_review manual__2024-11-03T04:55:16.752730+00:00 [queued]>
[2024-11-03T04:55:19.568+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-03T04:55:19.574+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): create_table_user_review> on 2024-11-03 04:55:16.752730+00:00
[2024-11-03T04:55:19.578+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=85) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-03T04:55:19.579+0000] {standard_task_runner.py:72} INFO - Started process 90 to run task
[2024-11-03T04:55:19.580+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'create_table_user_review', 'manual__2024-11-03T04:55:16.752730+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmpww_9cdon']
[2024-11-03T04:55:19.584+0000] {standard_task_runner.py:105} INFO - Job 27: Subtask create_table_user_review
[2024-11-03T04:55:19.608+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.create_table_user_review manual__2024-11-03T04:55:16.752730+00:00 [running]> on host bcc1ba72d809
[2024-11-03T04:55:19.659+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='create_table_user_review' AIRFLOW_CTX_EXECUTION_DATE='2024-11-03T04:55:16.752730+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-03T04:55:16.752730+00:00'
[2024-11-03T04:55:19.660+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-03T04:55:20.819+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-03T04:55:20.826+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-03T04:55:20.827+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=create_table_user_review, run_id=manual__2024-11-03T04:55:16.752730+00:00, execution_date=20241103T045516, start_date=20241103T045519, end_date=20241103T045520
[2024-11-03T04:55:20.872+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-03T04:55:20.886+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-03T04:55:20.887+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
